Name: Increase Top Bar Font Size - 18
Version: 1.3.1-1
Author: jmartino2011
Description: Increases the top bar font size to 18 (16 for the app name)

--- .orig/usr/lib/luna/system/luna-systemui/stylesheets/systemui.css
+++ /usr/lib/luna/system/luna-systemui/stylesheets/systemui.css
@@ -25,7 +25,7 @@
 .status-bar {
 		color: #fff;
 		font-weight: bold;
-		font-size: 14px;
+		font-size: 18px;
 		line-height: 25px;
 }
 
@@ -52,7 +52,7 @@
 }
 
 .clock {
-		font-size: 15px;
+		font-size: 18px;
 }
 
 #devicemenu {
@@ -73,7 +73,7 @@
 
 .appname-wrapper {
 		font-weight: bold;
-		font-size: 14px;
+		font-size: 16px;
 		line-height: 26px;
 		color: #fff;
 		margin: 0px 3px 0 -4px;
@@ -82,7 +82,7 @@
 
 .carrier-wrapper {
 		font-weight: bold;
-		font-size: 14px;
+		font-size: 18px;
 		line-height: 27px;
 		color: #fff;
 		max-width: 132px;
